### DISCUSSION TITLE
GitHub Education Application

### DISCUSSION THREAD

**Discussion by Krayzei** [community/community#143790] [GitHub Education]
### Hi everyone,

Hi everyone,

When I check my location, the application form does not go to the next step.  My browser gets a request for location permission which I approve; however,iIt just hangs there.  I've tried this with Edge and Vivaldi with the same result.  Does anyone have an idea to get to the next step?

![image](https://github.com/user-attachments/assets/62334d3a-aa7a-418f-bbe7-47396f8aaafb)

**Comment by the-matrixneo** [comment]
Hello👋 @Krayzei 
I have tried with Microsoft edge . You can simply turn on your location🗾 from setting and select the school from  drop-down box .

**Comment by the-matrixneo** [comment]
You can refer to👇👇 https://github.com/github-education-resources/Student-Developer-Pack-Current-Partners-FAQ 
I will help you 🚀

**Comment by lllHunterlllSV** [comment]
**Hello! 👋**

Thank you for reaching out! It seems like you're facing an issue with the location permission step while trying to submit your application. Here are a few things to check and try that could help resolve the issue:

- 🌐 **Browser Settings**: Sometimes, browsers block location permissions by default. Even if you approve the request, check if the browser has any settings preventing it. You can go into the **settings** (usually under Privacy & Security) and ensure location access is enabled for the site.
  
- 🔄 **Clear Cache and Cookies**: Try clearing your browser’s cache and cookies, as sometimes old data can cause the page to hang. After clearing, try refreshing the page and reattempting the process.

- 🖱 **Disable Extensions**: Browser extensions (like ad blockers or privacy tools) can sometimes interfere with the location request. Try disabling them temporarily and then reload the page.

- 📱 **Try Incognito/Private Mode**: Open a new **Incognito** (Chrome) or **Private** (Firefox/Edge) window and try again. This can help rule out issues related to saved data or cookies.

- 🌍 **Try a Different Network**: Sometimes, the issue may be related to your network’s configuration (e.g., VPNs or firewalls). If possible, try using a different network or disabling your VPN.

- 🚀 **Alternative Browsers**: You mentioned trying Edge and Vivaldi. You could also try **Chrome** or **Firefox**, as they tend to work well with most applications.

If these steps don’t resolve the issue, you can also try reaching out to [GitHub Support](https://support.github.com/) for more specific troubleshooting.

I hope one of these solutions helps, and best of luck getting through the process! 😊
